UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Helpers/FssApiClientTests.cs

```diff
@@ -197,6 +197,28 @@ public void DoesDownloadFile_Returns_OK()
             });
         }
 
+        [Test]
+        public void DoesDownloadFile_With_Client_That_Allow_Redirect_As_False_Returns_OK()
+        {
+            string Content = "";
+
+            HttpMessageHandler? messageHandler = FakeHttpMessageHandler.GetHttpMessageHandler(
+                               Content, HttpStatusCode.OK);
+
+            var httpClient = new HttpClient(messageHandler)
+            {
+                BaseAddress = new Uri("http://test.com")
+            };
+
+            A.CallTo(() => _fakeHttpClientFactory.CreateClient(A<string>.Ignored)).Returns(httpClient);
+
+            _fakeFssApiClient = new FssApiClient(_fakeHttpClientFactory);
+
+            Task<HttpResponseMessage>? result = _fakeFssApiClient.DownloadFile("http://test.com", _authToken);
```

Comment:
This doesn't need to be nullable (same for all the similar methods in this class).